File: app/src/main/java/com/example/starwars/model/Films.java
package com.example.starwars.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Films {
    @SerializedName("results")
    private List<Film> films;
    @SerializedName("total_pages")
    private int pages;

    public Films() {
    }

    public Films(List<Film> results) {
        this.films = results;
    }

    public List<Film> getFilms() {
        return films;
    }

    public int getPages() {
        return pages;
    }
}

File: app/src/main/java/com/example/starwars/model/Films.kt
package com.example.starwars.model

import com.google.gson.annotations.SerializedName
import com.example.starwars.model.Film

class Films {
    @SerializedName("results")
    var films: List<Film>? = null
        private set

    @SerializedName("total_pages")
    val pages = 0

    constructor() {}
    constructor(results: List<Film>?) {
        films = results
    }
}